src/platform/testing/unit/schemaform-utils.jsx

```diff
@@ -135,7 +164,14 @@ function printTree(node, level = 0, isLastChild = true, padding = '') {
   });
 }
 
+/**
+ * Gets the DOM node associated with the form tree passed in.
+ *
+ * @param {React.Element} form The root level of a rendered form
+ * @returns {object} An DOM node for the form, with added helper methods
+ */
 export function getFormDOM(form) {
+  // eslint-disable-next-line react/no-find-dom-node
```

Comment:
Didn't you override this already in `.eslintrc`?

Reply:
Yep, missed removing this afterward.